controller.go

```diff
@@ -222,6 +222,26 @@ func (c *controller) Zones() ([]Zone, error) {
 	return result, nil
 }
 
+// Pools implements Controller.
+func (c *controller) Pools() ([]Pool, error) {
```

Comment:
Please add a test to controller_test.go similar to the zone tests that are already there